modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/deploy/enums/DestroyScenario.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.service.deploy.enums;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.xpanse.modules.models.common.exceptions.UnsupportedEnumValueException;
+
+/**
+ * Destroy Scenario. The destroy scenario is used to set into TerraformResult and sent back to the
+ * client which sent the destroy request in what scenario.
+ */
+public enum DestroyScenario {
```

Comment:
if this enum is not exposed to customer, we must move this to the same place where "DeployTask" class exists.

Reply:
Moved.